examples/server_profiles.go

```diff
@@ -52,7 +56,7 @@ func main() {
 		InitialScopeUris:  *initialScopeUris,
 	}
 
-	err = ovc.SubmitNewProfile(server_profile_create_map)
+	err = ovc.SubmitNewProfile(server_profile_create_map, ignoreFlags...)
```

Comment:
Now the example seems to be running in golang without issue/

Reply:
@FalcoSuessgott  you can merge your PR and make the changes in terraform as well and raise a review.